Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/LogElementState.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Aquality.Selenium.Core.Elements
+{
+    /// <summary>
+    /// Logs element state.
+    /// </summary>
+    /// <param name="messageKey">Key of localized message to log.</param>
+    /// <param name="stateKey">Key of localized state to log.</param>
+    public delegate void LogElementState(string messageKey, string stateKey);
```

Comment:
Why do we need this delegate?

Reply:
to log element state, as it is stated by the name

Reply:
without such delegate I'll have to pass 4 additional parameters to each implementation of ElementStateProvider's constructor: ElementType, Name, LocalizationManager, LocalizedLogger.
Seems that a single delegate parameter is better solution. 
Do you have any suggestions how to make it better?